index.html

```diff
@@ -82,37 +82,37 @@ <h2>Profesional Art Printing Data</h2>
                 <li><a href="">Bootstrap</a></li>
                 <li><a href="">Ruby</a></li>
             </ul>
-            <button class="button" type="submit">See Project</button>
+            <button class="button"  type="submit">See Project</button>
```

Comment:
- [ ] I think it would be better if you stick with the  requirements as much as possible so kindly add all the projects/works section dynamicllay by js file so kindly remove all all the section from the html and ad it in js file.🔥💪🏻

![image](https://user-images.githubusercontent.com/58553711/180353840-c72f47de-9f72-4ac1-89cb-64e5f49a7f92.png)